app/views/events/index.slim

```diff
@@ -1,11 +1,13 @@
-h2 Events
--@events.each do |event|
-  = link_to event.name, event_path(event.id)
-  ul
-    li 
-      b = "On #{Date::DAYNAMES[event.start_date.wday]}"
-    li = event.starting_date
-    li = event.ending_date
-      li.fa.fa-user = "#{event.attendees.size}"
--if !current_user.nil?
-  = link_to 'New Event', new_event_path
+ul.Grid
+  -@events.each do |event|
+    .Grid-item
```

Comment:
I think it would make more sense, semantically, for the `li` to be attached to `Grid-item` instead of `Event`.
What do you think?